assets/airports/kstl.json

```diff
@@ -493,14 +493,14 @@
             "icao": "EEMOS4",
             "name": "Emose Four",
             "rwy": {
-                "6": ["#064"],
-                "11": [["FUNKE", "A25+"], "BEART", "GMSKI"],
-                "12L": [["FUNKE", "A25+"], "BEART", "GMSKI"],
-                "12R": [["FUNKE", "A25+"], "BEART", "GMSKI"],
-                "24": ["#244"],
-                "29": [["DSAPP", "A25+"], "CONRR"],
-                "30L": [["WOLTR", "A25+"], "CONRR"],
-                "30R": [["WOLTR", "A25+"], "CONRR"]
+                "KSTL6": ["#064"],
+                "KSTL11": [["FUNKE", "A25+"], "BEART", "GMSKI"],
```

Comment:
I'm assuming that the climb to 1120' is not being simulated?

Reply:
Since airport elevation is 618ft, this just means aircraft have to climb 500ft before turning, which the sim already does anyway so no intermediate fix is required